jobs/mongodb_migration/src/mongodb_migration/collector.py

```diff
@@ -276,4 +278,17 @@ def get_migrations(self) -> list[Migration]:
                 version="20231106193200",
                 description="add 'partial', 'num_rows' and 'num_bytes' fields for 'split-duckdb-index' cache records",
             ),
+            MigrationDeleteJobsByStatus(
+                status_list=["success", "error", "cancelled"],
+                version="20231201074900",
+                description="delete jobs with success, error and cancelled status",
+            ),
+            MigrationQueueDeleteTTLIndex(
+                version="20231201112000",
+                description="delete the TTL index on the 'finished_at' field in the queue database",
+                field_name="finished_at",
+            ),
+            MigrationRemoveFieldFromJob(
```

Comment:
same kind of comment: we will delete the field, but: as the current pods will still finish jobs meanwhile, we will have to run the migration again. Not sure how to do it in two PRs...

Anyway: we can just try and run twice (and fix afterwards if something broke)

Reply:
(and same comment for the deletion of jobs with a finished status: they will still exist).

Maybe the best strategy would be:
- a PR that removes the "finished_at" field + the index, in the code
- a PR that only runs the three migrations

Reply:
I created the following PRs that should be merged/deployed in this order:
https://github.com/huggingface/datasets-server/pull/2178 (removing  "finished_at" field + the index, in the code
https://github.com/huggingface/datasets-server/pull/2179 (migrations)
and then https://github.com/huggingface/datasets-server/pull/2175 (this)